src/gateway_mfr_worker.erl

```diff
@@ -288,7 +292,7 @@ check_miner_key(Pid) ->
 %%
 
 gen_compact_key(Pid, Slot) ->
-    gen_compact_key(Pid, Slot, 100).
+    gen_compact_key(Pid, Slot, 200).
```

Comment:
Why did you have to double the number of retries to 200? That seems like there's an issue with the part you are using if it needs this many retries to generate a private key

Reply:
As test by my customer, they find many failure chip, which can't pass the private key testing as not compact key. So we can increase to retry 200 times to gen a compact private key.

Reply:
This usually implies a hardware integration issue. We used to have the same problems on the original Helium hotspot and it turned out to be either a badly configured driver or i2c bus issues